src/renderer/dx/precomp.h

```diff
@@ -4,9 +4,11 @@
 #pragma once
 
 // This includes support libraries from the CRT, STL, WIL, and GSL
+#define BLOCK_TIL // We want to include it later, after DX.
 #include "LibraryIncludes.h"
 
 #include <windows.h>
+#include <winmeta.h>

```

Comment:
?

Reply:
Needed it for the flags on the `TraceLogging` macros. Somehow the `WINEVENT_*` defines weren't all visible. This makes them visible without pulling in a lot of other stuff.